relayer/chains/icon/listener.go

```diff
@@ -88,11 +89,12 @@ func (p *Provider) Listener(ctx context.Context, lastProcessedTx providerTypes.L
 						}
 						for _, msg := range msgs {
 							p.log.Info("Detected eventlog",
-								zap.Uint64("height", msg.MessageHeight),
-								zap.String("target_network", msg.Dst),
 								zap.Uint64("sn", msg.Sn.Uint64()),
-								zap.String("tx_hash", v.Hash.String()),
+								zap.Uint64("req-id", msg.ReqID.Uint64()),
```

Comment:
Will every message detected here have reqId ?

Reply:
> Will every message detected here have reqId ?

No. I think it should not be a problem printing <nil> in the log.